lab2.js

```diff
@@ -139,3 +293,4 @@ function sumSort(arrayOfArrays) {
 // ./node_modules/.bin/grunt jshint
 // ./node_modules/.bin/grunt jscs
 //*********************************************************
+//I did those things! Is it okay if we customize jscs? (I'll probably just ask you this)
```

Comment:
Yes...as long as you're not customizing it so anything passes. :) I often use a different combination of settings - check out https://github.com/stephanielingwood/should-I-let-the-cat-outside/blob/master/.jscsrc if you want a look at my .jscsrc. It's basically the airbnb preset with some alterations. Gabe Medrash also has a similar version here: https://github.com/GabeMedrash/angular-mean-median-mode/blob/master/.jscsrc